scripts/build_android_libtorch_and_pytorch_android.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+# Must be run from the root directory of pytorch repo
+set -eux
+
+PYTORCH_DIR=$(pwd)
+PYTORCH_ANDROID_DIR=$PYTORCH_DIR/android
+WORK_DIR=$(pwd)
+
+echo "PYTORCH_DIR:$PYTORCH_DIR"
+echo "WORK_DIR:$WORK_DIR"
+
+echo "ANDROID_HOME:$ANDROID_HOME"
+if [ ! -z "$ANDROID_HOME" ]; then
+  echo "ANDROID_HOME not set; please set it to Android sdk directory"
+fi
+
+if [ ! -d $ANDROID_HOME ]; then
+  echo "ANDROID_HOME not a directory; did you install it under $ANDROID_HOME?"
+  exit 1
+fi
+
+echo "ANDROID_NDK:$ANDROID_NDK"
+if [ ! -z "$ANDROID_NDK" ]; then
+  echo "ANDROID_NDK not set; please set it to Android sdk directory"
+fi
+
+if [ ! -d $ANDROID_NDK ]; then
+  echo "ANDROID_NDK not a directory; did you install it under $ANDROID_NDK?"
+  exit 1
+fi
+
+GRADLE_PATH=gradle
+GRADLE_NOT_FOUND_MSG="Unable to find gradle, please add it to PATH or set GRADLE_HOME"
+
+if [ ! -x "$(command -v gradle)" ]; then
+  if [ -z "$GRADLE_HOME" ]; then
+    echo GRADLE_NOT_FOUND_MSG 
+    exit 1
+  fi
+  GRADLE_PATH=$GRADLE_HOME/bin/gradle
+  if [ ! -f "$GRADLE_PATH" ]; then
+    echo GRADLE_NOT_FOUND_MSG 
+    exit 1
+  fi
+fi
+echo "GRADLE_PATH:$GRADLE_PATH"
+
+ABI_ARM32=armeabi-v7a
+ABI_ARM64=arm64-v8a
+ABI_X86=x86
+ABI_X86_64=x86_64
+
+ABIS_LIST="$ABI_X86 $ABI_X86_64 $ABI_ARM32 $ABI_ARM64"
```

Comment:
nit: why don't you directly use the ABI names inline as they are only used once here?

Reply:
Thanks, I will replace them here.
Initially I had an idea to simplify specification of that list in the way:
1000 - x86 (param=8)
1100 - x86, x86_64 (param=12)
0010 - armeabi-v7a (param=2)
0001 - arm64-v8a (param=1)
1111 - all abis (param=15)

But after that decided that mostly it will be used x86 or all :)

Reply:
inlined